stochastic_physics/stochastic_physics_wrapper.F90

```diff
@@ -282,6 +282,7 @@ subroutine stochastic_physics_wrapper (GFS_Control, GFS_Data, Atm_block, ierr)
                   endif
 
                   select case (trim(GFS_Control%lndp_var_list(v)))
+                  ! DH* is this correct? shouldn't this be slc ?
```

Comment:
This may be an actual bug. Somebody needs to verify. But I want to leave it as is because it may change the results - this PR is supposed to be b4b!